drizzlepac/hlautils/astrometric_utils.py

```diff
@@ -70,57 +68,35 @@
 
 MODULE_PATH = os.path.dirname(inspect.getfile(inspect.currentframe()))
 VEGASPEC = os.path.join(os.path.dirname(MODULE_PATH),
-                        'data','alpha_lyr_stis_008.fits')
+                        'data', 'alpha_lyr_stis_008.fits')
 
 __all__ = ['create_astrometric_catalog', 'compute_radius', 'find_gsc_offset',
-            'extract_sources', 'find_hist2d_offset', 'generate_source_catalog',
-            'classify_sources', 'countExtn']
+           'extract_sources', 'find_hist2d_offset', 'generate_source_catalog',
+           'classify_sources', 'countExtn']
```

Comment:
The entry for 'countExtn' is invalid here, since that function was removed from this module.